.github/workflows/conformance-e2e.yaml

```diff
@@ -369,7 +369,7 @@ jobs:
             if [ "${{ matrix.ipv6 }}" == "false" ]; then
               IP_FAM="ipv4"
             fi
-            ./contrib/scripts/kind.sh --xdp "" 3 "" "" "${{ matrix.kube-proxy }}" \$IP_FAM
+            ./contrib/scripts/kind.sh --xdp --secondary-interface "" 3 "" "" "${{ matrix.kube-proxy }}" \$IP_FAM
```

Comment:
Maybe let's call it `--secondary-network`?

Reply:
Done